src/compass/s1_geocode_slc.py

```diff
@@ -20,6 +21,39 @@
 from compass.utils.yaml_argparse import YamlArgparse
 
 
+def init_backscatter_dataset(h5_root, polarization, geo_grid):
+    '''
+    Create and allocate dataset for isce.geocode.geocode_slc to write to
+
+    h5_root: h5py
+        Root to CSLC
+    polarization: str
+        Polarization to be used as dataset name
+    geo_grid: isce3.product.GeoGridParameters
+        Geogrid out output
+    '''
+    # bs : short for backscatter
+    bs_group = h5_root.require_group('complex_backscatter')
+
+    # Data type
+    ctype = h5py.h5t.py_create(np.complex64)
+    ctype.commit(h5_root['/'].id, np.string_('complex64'))
+
+    shape = (geo_grid.length, geo_grid.width)
+    bs_ds = bs_group.create_dataset(polarization, dtype=ctype, shape=shape)
+
+    descr = f'Geocoded SLC image ({polarization})'
+    bs_ds.attrs['description'] = np.string_(descr)
+
+    long_name = f'geocoded single-look complex image {polarization}'
```

Comment:
```suggestion
    long_name = f'{polarization} geocoded single-look complex image '
```
Usually, we put the polarization channel first